fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -657,6 +787,12 @@ def provider_ids
 
     TWO_FACTOR_ENV_VARIABLE = "FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD"
 
+    # Returns whether altool should be used or ItunesTransporter should be used
+    def should_use_altool(upload, use_shell_script)
```

Comment:
In Ruby convention, methods return boolean should end with `?`. like `should_use_altool?`

Reply:
done! 
5bd7d3f